CHANGELOG.md

```diff
@@ -4,6 +4,26 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 4.0.0
+### Added
+* Agent: Added common source coverage format. [#1403](https://github.com/microsoft/onefuzz/pull/1403)
+* Service: Added class to store and retrieve rules associated with an API endpoint. This supports the ability to control who has access to an API. [#1420](https://github.com/microsoft/onefuzz/pull/1420)
+* Service: Support for creation of NSG during deployment allowing restricted access to the scaleset and repro VM's. [#1331](https://github.com/microsoft/onefuzz/pull/1331), [#1340](https://github.com/microsoft/onefuzz/pull/1340), [#1358](https://github.com/microsoft/onefuzz/pull/1358), [#1385](https://github.com/microsoft/onefuzz/pull/1385), [#1393](https://github.com/microsoft/onefuzz/pull/1393), [#1395](https://github.com/microsoft/onefuzz/pull/1395), [#1400](https://github.com/microsoft/onefuzz/pull/1400), [#1404](https://github.com/microsoft/onefuzz/pull/1404), [#1406](https://github.com/microsoft/onefuzz/pull/1406), [#1410](https://github.com/microsoft/onefuzz/pull/1410)
+* Service: Disable guest account access by default when creating the default service principal during deployment. [#1425](https://github.com/microsoft/onefuzz/pull/1425)
```

Comment:
```suggestion
* Service: Disabled guest account access by default when creating the default service principal during deployment. [#1425](https://github.com/microsoft/onefuzz/pull/1425)
```

Reply:
The past tense doesn't sound right to me.

Reply:
I suggested this since we use past tense _everywhere_ else in this changelog, but I think you're right in this case, given how the verb is being used (we're describing a policy, vs. a past-tense change to the code like "added" or "fixed").

Reply:
Maybe "Guest account access disabled by default..."?